lib/libxdp/libxdp.map

```diff
@@ -39,4 +39,13 @@ LIBXDP_1.0.0 {
 };
 
 LIBXDP_1.1.0 {
+	        xsk_setup_xdp_prog;
+		xsk_socket__create;
+		xsk_socket__create_shared;
+		xsk_socket__delete;
+		xsk_socket__fd;
+		xsk_socket__update_xskmap;
+		xsk_umem__create;
+		xsk_umem__delete;
+		xsk_umem__fd;
```

Comment:
So since we already tagged 1.1.0, these should really be in LIBXDP_1.2.0. I guess since there are no symbols in 1.1.0, you can just rename the section...